pkgs/development/libraries/gtk/4.x.nix

```diff
@@ -277,5 +283,6 @@ stdenv.mkDerivation rec {
     maintainers = teams.gnome.members ++ (with maintainers; [ raskin ]);
     platforms = platforms.all;
     changelog = "https://gitlab.gnome.org/GNOME/gtk/-/raw/${version}/NEWS";
+    pkgConfigModules = ["gtk4"];
```

Comment:
This package has more `.pc` files within the `gtk4.dev` output.

I added them in eldritch-cookie/nixpkgs#1.